lemur/certificates/models.py

```diff
@@ -297,6 +296,7 @@ def location(self):
     def distinguished_name(self):
         return self.parsed_cert.subject.rfc4514_string()
 
+    """
```

Comment:
I would add a comment here, maybe along the lines of to be removed, as the functionality is provided via key_type column.